client/jetpack-onboarding/main.jsx

```diff
@@ -74,10 +74,11 @@ export default connect(
 	( { siteId, ...stateProps }, { recordTracksEvent: recordTracksEventAction }, ownProps ) => ( {
 		siteId,
 		...stateProps,
-		recordJpoEvent: event =>
+		recordJpoEvent: ( event, additionalProperties ) =>
```

Comment:
Super nit: I'd call it `extraProperties.`

Reply:
I was influenced by JSON Schema when choosing the name here 😉  see http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.5.6